client/src/components/room/scaffold/Voters.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Wrapper } from '@components/room/scaffold/Voters.style';
+import { VoterIcon } from '@components/icons';
+import { COLOR } from '@constant/style';
+
+type VotersPropType = {
+  total: number;
+  approves: number;
+  rejects: number;
+};
+
+const Voters: React.FC<VotersPropType> = ({ total, approves, rejects }) => {
+  return (
+    <Wrapper>
+      {Array.from({ length: total }, (_, index) => {
+        let color = COLOR.body;
+        if (index < approves) color = COLOR.titleActive;
+        if (index > total - rejects - 1) color = COLOR.error;
```

Comment:
```suggestion
const isVoteApprove = index < approves;
const isVoteReject = index > total - rejects - 1;
        if (isVoteApprove) color = COLOR.titleActive;
        if (isVoteReject) color = COLOR.error;
```
요런식으로 이름을 붙여주면 처음 봤을 때 좀 더 빨리 이해할 수 있을 것 같아요!

Reply:
오... 리뷰에 Suggest 기능 처음봐요!
확실하게 아랫 부분이 더 명확하게 보이는 것 같아요,, 바로 적용하겠습니다